packages/dx-react-grid/src/plugins/drag-drop-provider.tsx

```diff
@@ -53,18 +54,21 @@ class DragDropProviderBase extends React.PureComponent<DragDropProviderProps, Dr
           {payload && (
             <TemplateConnector>
               {({ columns }) => (
-                <Container
-                  clientOffset={clientOffset!}
-                >
-                  {getTargetColumns(payload, columns)
-                    .map(column => (
-                      <Column
-                        key={column.name}
-                        column={column}
-                      />
-                    ))
-                  }
-                </Container>
+                createPortal(
+                  <Container
+                    clientOffset={clientOffset!}
+                  >
+                    {getTargetColumns(payload, columns)
+                      .map(column => (
+                        <Column
+                          key={column.name}
+                          column={column}
+                        />
+                      ))
+                    }
+                  </Container>,
+                  document.body,
```

Comment:
This code doesn't work on SSR. We should avoid using the `document` variable in the render method.